src/main/java/com/ericsson/ei/notifications/EmailSender.java

```diff
@@ -95,10 +96,10 @@ public MimeMessage prepareEmailMessage(String recipients, String mapNotification
      * at application.proporties are empty.
      * */
     private void setDefaultValuesEmailSenderSubject() {
-       if (subject.isEmpty()) {
+       if (StringUtils.isEmpty(subject) == true) {
             subject = "Email Subscription Notification";
         }
-       if (sender.isEmpty()) {
+       if (StringUtils.isEmpty(sender) == true) {
```

Comment:
No need for the "== true". The StringUtils.isEmpty(sender) gives a boolean result which can be used as is, in the if statement.